ts/utils/paymentMethod.ts

```diff
@@ -68,6 +68,9 @@ export const getTitleFromCard = (
 export const getBancomatAbiIconUrl = (abi: string) =>
   `${contentRepoUrl}/logos/abi/${abi}.png`;
 
+export const getPayPalPspIconUrl = (abi: string) =>
```

Comment:
What do you think about unify `getBancomatAbiIconUrl` and `getPayPalPspIconUrl` since they do the same thing?

Reply:
I prefer to keep them separated even if they are equal.

First to have a dedicated name for each payment method and then to avoid possible bugs when you change it: if the function is one for more methods, you (aka developer) could change it to have an expected behavior for a specific method but at the same time you could break it for the others one

Reply:
ok, sounds good!